br/pkg/restore/split_client.go

```diff
@@ -3,624 +3,14 @@
 package restore
```

Comment:
Can we remove this file? Since most logic is moved to the `split/` directory.

Reply:
I have moved all split_client.go logic into split/split_client.go doc.

Reply:
Done